style.css

```diff
@@ -0,0 +1,341 @@
+* {
```

Comment:
Kindly implement your styling based on the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided.

Example;
Your mobile design:
<img width="587" alt="image" src="https://user-images.githubusercontent.com/31897434/216038090-3bd5ef7e-66bc-4552-b7fe-f574c9dcab36.png">

The provided mobile design:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/31897434/216038269-edf78ec1-bbf8-4e97-8d23-4dd5f4d58fa2.png">